tests/src/Functional/DatasetBTBTest.php

```diff
@@ -29,23 +29,83 @@ class DatasetBTBTest extends BrowserTestBase {
     'field',
     'harvest',
     'metastore',
+    'metastore_search',
     'node',
+    'search_api',
   ];
 
   /**
    * {@inheritdoc}
    */
-  protected $defaultTheme = 'starterkit_theme';
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $strictConfigSchema = FALSE;
```

Comment:
@paul-m and I discussed this. This setting is related to change record https://www.drupal.org/node/3362879 and if it is left true, the test will fail due to poorly defined config schema in contrib modules as well. Suggest adding a comment referencing this!